common/views/components/ModalMoreFilters/ModalMoreFilters.tsx

```diff
@@ -0,0 +1,359 @@
+              })}
+            >
+              {subjectsFilter.length > 0 && (
+                <RadioGroup
+                  name="subjects.label"
+                  selected={subjectsInUrl}
+                  onChange={changeHandler}
+                  options={getAggregationRadioGroup(subjectsFilter, 'desktop')}
+                />
+              )}
+            </div>
+          </Space>
+        </FilterSection>
+      )}
+      {filtersToShow.includes('genres') && (
+        <FilterSection>
+          <h3 className="h3">Genres</h3>
+          <Space as="span" h={{ size: 'm', properties: ['margin-right'] }}>
+            <div
+              className={classNames({
+                'no-margin no-padding plain-list': true,
+              })}
+            >
+              {subjectsFilter.length > 0 && (
+                <RadioGroup
+                  name="genres.label"
+                  selected={genresInUrl}
+                  onChange={changeHandler}
+                  options={getAggregationRadioGroup(genresFilter, 'mobile')}
+                />
+              )}
+            </div>
+          </Space>
+        </FilterSection>
+      )}
+      {filtersToShow.includes('languages') && (
+        <FilterSection>
+          <h3 className="h3">Languages</h3>
+          <Space as="span" h={{ size: 'm', properties: ['margin-right'] }}>
+            <ul
+              className={classNames({
+                'no-margin no-padding plain-list': true,
+              })}
+            >
+              {languagesFilter.length > 0 &&
+                languagesFilter.map(language => {
+                  const isChecked = languagesInUrl.includes(language.data.id);
+                  return (
+                    (language.count > 0 || isChecked) && (
+                      <Space
+                        as="li"
+                        v={{ size: 'l', properties: ['margin-bottom'] }}
+                        key={`desktop-${language.data.id}`}
+                      >
+                        <CheckboxRadio
+                          id={`desktop-${language.data.id}`}
+                          type={`checkbox`}
+                          text={`${language.data.label} (${language.count})`}
+                          value={language.data.id}
+                          name={`languages`}
+                          checked={isChecked}
+                          onChange={changeHandler}
+                          ariaLabel={searchFilterCheckBox(language.data.label)}
+                        />
+                      </Space>
+                    )
+                  );
+                })}
+            </ul>
+          </Space>
+        </FilterSection>
+      )}
+    </>
+  );
+};
+
+const ModalMoreFilters: FunctionComponent<MoreFiltersProps> = ({
+  id,
+  showMoreFiltersModal,
+  setMoreFiltersModal,
+  openMoreFiltersButtonRef,
+  filtersToShow,
+  aggregations,
+  changeHandler,
+  languagesInUrl,
```

Comment:
Could we keep this naming to not include anything to do with the URL.
It is derived from the URL at the beginning of the app, but for this components, we are just interested in the values.

I would just call these `languages` or `selectedLanguages` and have the aggregations called exactly that, `languagesAggregations`.